bin/control.sh

```diff
@@ -31,24 +31,33 @@ fi
 #
 # Import common functions.
 #
-if [ "${IGNITE_HOME:-}" = "" ];
-    then IGNITE_HOME_TMP="$(dirname "$(cd "$(dirname "$0")"; "pwd")")";
-    else IGNITE_HOME_TMP=${IGNITE_HOME};
+if [ "${IGNITE_HOME:-}" = "" ]; then
+    IGNITE_HOME_TMP="$(dirname "$(cd "$(dirname "$0")"; pwd)")"
+else IGNITE_HOME_TMP=${IGNITE_HOME}
 fi
 
 #
 # Set SCRIPTS_HOME - base path to scripts.
 #
 SCRIPTS_HOME="${IGNITE_HOME_TMP}/bin"
 
-source "${SCRIPTS_HOME}"/include/functions.sh
-source "${SCRIPTS_HOME}"/include/jvmdefaults.sh
+source "${SCRIPTS_HOME}/include/functions.sh"
+source "${SCRIPTS_HOME}/include/jvmdefaults.sh"
 
 #
-# Discover path to Java executable and check it's version.
+# Discover path to Java executable and check its version.
 #
 checkJava
 
+#
+# Determine Java version function
+#
+determineJavaVersion() {
```

Comment:
Why do we need this function? `checkJava` that is called 2 lines above should store Java version in `version` variable, can we reuse it?

Reply:
The `determineJavaVersion()` function was added following Slava's guidance to make the Java version check explicit in `control.sh`. This was intended to manage JVM options across Java versions effectively, especially for compatibility with different Java behaviours.

Reply:
I didn't find any comments from Slava with such request. Maybe there were a bit of misunderstanding. I believe that this function along with `JAVA_VERSION` variable is redundant.

Reply:
Hi @solveme, now I couldn't find it either. There must have been a misunderstanding. The adding of that `determineJavaVersion` function was indeed redundant. I've removed the `determineJavaVersion `function and its usage from both `control.sh` and `control.bat`.
Thanks.